presentation/src/main/java/com/pocs/presentation/model/post/BasePostEditUiState.kt

```diff
@@ -12,6 +12,7 @@ interface BasePostEditUiState {
     val onContentChange: (String) -> Unit
     val onCategoryChange: (PostCategory) -> Unit
     val onSave: suspend () -> Result<Unit>
+    val showChips : Boolean
```

Comment:
음 속성보다는 getter로 하는게 간결하고 좋습니다. 이렇게 하면 이 인터페이스를 구현한 객체인 PostCreateUiState와 PostEditUiState에서 overriding 할 필요가 없기 때문이죠. 또한 추가로 uistate 객체를 생성할때 argument로 전달할 필요도 없고요.
```suggestion
    val showChips get() = category != PostCategory.QNA
```